ci/check-update-build-webide.sh

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env bash
+# Copyright (c) 2019 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+set -euo pipefail
+
+GITHUB_API_TOKEN=${GITHUB_API_TOKEN:-""}
+GOOGLE_APPLICATION_CREDENTIALS_CONTENT=${GOOGLE_APPLICATION_CREDENTIALS_CONTENT:-""}
+
+#sets LATEST_SDK_WITH_TIME
+getLatestSdk() {
+  local TMP_FILE=$(mktemp)
+  local JQ_COMMAND='. | map(select(.prerelease == false)) | map([.tag_name,.published_at]) | map(.[0] + "," + .[1] + "," + (.[1]|fromdate|tostring))[]'
+  if [ -n "$GITHUB_API_TOKEN" ]; then
+    $(curl -H "Authorization: $GITHUB_API_TOKEN" https://api.github.com/repos/digital-asset/daml/releases -s > "$TMP_FILE")
+  else
+    $(curl https://api.github.com/repos/digital-asset/daml/releases -s > "$TMP_FILE")
+  fi
+  local RELEASES=$(jq -r "$JQ_COMMAND" "$TMP_FILE")
+  local LAST_COMMAND_RESULT=$?
+  if [ $LAST_COMMAND_RESULT -eq 2 ]; then
+    echo "ERROR: could not read daml releases `cat $TMP_FILE`"
+    return 1
+  fi
+  LATEST_SDK_WITH_TIME=$(echo $RELEASES | awk '{if(substr($1,1,1)=="v") print substr($1,2); else print substr($1,1);}')
```

Comment:
The final script at the end would be easier to follow if your functions returned their results rather than storing them in global variables.